src/TLuaInterpreter.cpp

```diff
@@ -4460,12 +4460,13 @@ int TLuaInterpreter::getImageSize(lua_State* L)
         return warnArgumentValue(L, __func__, QStringLiteral("bad argument #1 value (image location cannot be an empty string)"));
     }
 
-    if (auto size = mudlet::self()->getImageSize(imageLocation)) {
-        lua_pushnumber(L, size->width());
-        lua_pushnumber(L, size->height());
-        return 2;
-    } 
-    return warnArgumentValue(L, __func__, QStringLiteral("couldn't retrieve image size, is the location '%1' correct?").arg(imageLocation));
+    auto size = mudlet::self()->getImageSize(imageLocation);
+    if (!size) {
```

Comment:
You can do this in one line: https://skebanga.github.io/if-with-initializer/

```suggestion
    if (auto size = mudlet::self()->getImageSize(imageLocation); !size) {
```

Reply:
That is actually quite interesting!
I saw lots of possible spots to improve code with this already before.. 😄

Reply:
Yeah!

But I was wrong in this case because I don't think `size` will be available outside of the if/elseif/else :D

Reply:
You're right, it would be limited. And given I split the else-clause out of it, that won't fly anymore. So will not commit this.